@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -202,6 +194,24 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
             }
             virtualFocus.moveFocusUp();
           }
+        } else if (e.key === "Home") {
+          e.preventDefault();
+          virtualFocus.moveFocusToTop();
+        } else if (e.key === "End") {
+          e.preventDefault();
+          if (virtualFocus.activeElement === null || !isListOpen) {
+            toggleIsListOpen(true);
+          }
+          virtualFocus.moveFocusToBottom();
+        } else if (e.key === "PageUp") {
+          e.preventDefault();
```

Comment:
```suggestion
          e.preventDefault();
          if (virtualFocus.isFocusOnTheTop()) {
            toggleIsListOpen(false);
          }
```

Funket som bare det! (ref det du kommenterte om), var copy paste fra kode over 😎. Tror jeg liker at det er lik behaviour på tvers av alle pagineringsmuligheter 🤔.